__tests__/specs/core/feature_flags.cy.ts

```diff
@@ -0,0 +1,28 @@
+import { LabelStudio } from '@heartexlabs/ls-test/helpers/LSF';
+
+describe('Feature Flags', () => {
+  it('can set feature flags on the global object', () => {
+    const flagName = 'customFeatureFlag';
+
+    cy.visit('/');
+
+    LabelStudio.setFeatureFlags({
+      [flagName]: true,
```

Comment:
I'd check two flags with different values to be sure that this is not a default value in use.
```suggestion
      [enabledFlag]: true,
      [disabledFlag]: false,
```

Reply:
fair point. added your suggestion + added better helper for testing this stuff